scripts/zones/Grand_Palace_of_HuXzoi/mobs/Ixaern_MNK.lua

```diff
@@ -8,53 +8,94 @@ require("scripts/globals/status")
 -----------------------------------
 local entity = {}
 
+local bracerMode = function(mob, qnAern1, qnAern2)
+    local mobID = mob:getID()
+    mob:useMobAbility(690) -- Hundred Fists
+    if qnAern1:isAlive() then qnAern1:useMobAbility(692) end -- Chainspell
+    if qnAern2:isAlive() then qnAern2:useMobAbility(689) end -- Benediction
+    mob:addMod(xi.mod.ATT, 200)
+
+    for i = mobID+1, mobID+2 do
+        local pet = GetMobByID(i)
+        if pet:isSpawned() then
+            pet:AnimationSub(2)
+            pet:addMod(xi.mod.ATT, 200)
+            pet:setMod(xi.mod.DELAY, 2100)
+        end
+    end
+    -- slightly delay adding local var to avoid adding bracers to Ix'Mnk while Hundred Fists is still active.
+    mob:timer(3000, function(mobArg)
```

Comment:
I would rather we set vars each for both 2hrs it track them each than use timer, as onMobFight will continue to call this helper before the variable is set here and it is _possible_ to get edge cases seemingly at random where mob Ai queues up the moves repeatedly. onMobFight will exec every 400ms, slightly below half a second. It may work as you have it during your testing could be safer. I would still keep the bracer variable, but use the 2hr vars when deciding to set it if that makes sense.

Reply:
I did this in the original implementation. (Wings side) And it always just still two hour'd, wings just didn't update fast enough to catch the local var in time.

Reply:
I am apparently illterate today, read BracerMode and enableBracers as same variable.